.github/workflows/lint.yml

```diff
@@ -28,6 +28,7 @@ jobs:
               npm run lint
           fi
       - uses: taiga-family/ci/actions/auto/push@v1.103.0
+        continue-on-error: true
```

Comment:
Is this an expected change?

Reply:
fixed after https://github.com/taiga-family/taiga-ui/pull/10006